dockers/skylab/HCA_post_processing/test_optimus_HCA_loom_merge.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+from optimus_HCA_loom_merge import combine_loom_files
+import loompy
+import numpy as np
+import os
+import re
+from shutil import copyfile
+
+_test_data_dir = os.path.join(os.path.split(__file__)[0], "testdata/")
+def test_combine_loom_files(tmpdir):
+    import os
+    print('Test to merge three loom files')
+    # original file 
+    looma = os.path.join(_test_data_dir, "a.loom")
+    loomb = os.path.join(_test_data_dir, "b.loom")
+    loomc = os.path.join(_test_data_dir, "c.loom")
+    
+    # make a copy of these in the tmp file to avoid being overwritten
+    # during reapeated pytest runs on pytest
+    tmplooma = tmpdir.join("a.loom")
```

Comment:
No need to copy the files if we no longer change the original loom file in-place

Reply:
removed the steps to create temporary files